components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/SAMLSSOAuthenticator.java

```diff
@@ -294,6 +294,10 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
             stateInfoDO.setSpNameQualifier(spNameQualifier);
             context.setStateInfo(stateInfoDO);
 
+            //include the saml index with the federated idp in the authentication context
+            context.setProperty(context.getExternalIdP().getIdentityProvider().getIdentityProviderName(),
```

Comment:
write some mechanism to identify and add a space (and capital letter in the beginning) before the comment